pkg/daemon/ceph/client/filesystem.go

```diff
@@ -170,10 +170,10 @@ func AddDataPoolToFilesystem(context *clusterd.Context, clusterInfo *ClusterInfo
 	args := []string{"fs", "add_data_pool", name, poolName}
 	_, err := NewCephCommand(context, clusterInfo, args).Run()
 	if err != nil {
-		// Reef disallows calling add_data_pool for a pool that has already
+		// Squid (and later) disallows calling add_data_pool for a pool that has already
 		// been added, so ignore the error code.
 		// Previous releases do not return an error when an existing data pool is added.
-		if clusterInfo.CephVersion.IsAtLeastReef() {
+		if clusterInfo.CephVersion.IsAtLeastSquid() {
```

Comment:
No longer necessary to check for the version here, since all supported versions would have the new behavior.
```suggestion
```